### DISCUSSION TITLE
23.05分支云编译报错

### DISCUSSION THREAD

**Discussion by Pacio** [immortalwrt/immortalwrt#1495] [Q&A]
![image](https://github.com/user-attachments/assets/ff70584f-6c69-44cc-a6b2-638c95d917a9)
最新源码云编译报错，设备是x86/64

**Comment by 1715173329** [comment]
你配置问题。

**Reply by Pacio** [reply]
> 你配置问题。

默认配置没动，就选了几个插件

**Comment by Pacio** [comment]
![image](https://github.com/user-attachments/assets/ff287f3e-cdfd-45ce-b37d-a0b9b947b0dc)
好像不是配置的问题，就是缺少一些依赖，上个月我云编译没遇到这些问题啊~
